src/WebComponents/build/boot-lite.js

```diff
@@ -37,14 +37,12 @@
       }
     }
     // log flags
+    flags.log = {};
```

Comment:
This isn't a behavior preserving change. With this change, `flags.log` will always be defined and `flags.log.split` will always be undefined, so the if body will never be executed.